packages/nextjs-demo/src/app/agent/page.tsx

```diff
@@ -242,6 +242,13 @@ class ChatManager {
   }
 }
 
+const MenuItem: React.FC<{ name: string; price: number }> = ({ name, price }) => (
```

Comment:
If we cared about accessibility for this demo, you'd want to stick to the semantic markup (`ul` and `li`)s.